columbo/_interaction.py

```diff
@@ -32,6 +33,8 @@ class _Sentinel(Enum):
 _NOT_GIVEN = _Sentinel.A
 Possible = Union[T, _Sentinel]
 
+VALIDATOR_UPGRADE_DOCS_LINK = "https://wayfair-incubator.github.io/columbo/latest/usage-guide/validators/#upgrading-validator-structure"
```

Comment:
So this is tricky. `/latest/` if the docs at the time of the most recent release. So when this gets merged, the new section would be on `/dev/` until v0.10.0 is released. However, in the future when support is removed, this section could be removed. So `/latest/` would no longer be valid.

I think the best would be to point to `/0.10.0/`. It won't be valid now, but it will be valid once the version is released. Which is when people might start seeing the warning.

Reply:
Perfect, thanks!